times.go

```diff
@@ -25,28 +25,50 @@ func Time(then time.Time) string {
 	return RelTime(then, time.Now(), "ago", "from now")
 }
 
-var magnitudes = []struct {
+type Seconds int64
```

Comment:
I agree, I would try to use `time.Duration` and convert to seconds if/when needed.